Source/controls/game_controls.cpp

```diff
@@ -277,7 +277,7 @@ bool GetGameAction(const SDL_Event &event, ControllerButtonEvent ctrlEvent, Game
 					CalcViewportGeometry();
 				}
 #else
-			    // Not mapped. Reserved for future use.
+			                                // Not mapped. Reserved for future use.
 #endif
 				return true;

```

Comment:
???

Reply:
No idea what confused clang-format here but changing the comment from `//` to `/* */` seems to have fixed it.